Homework_2.0.py

```diff
@@ -17,38 +17,16 @@ def __init__(self, name, position):
         self.position = self.check_ascii(position)
 
     def check_ascii(self, param):
-        if type(param) == str:
-            if param.isascii():
-                return param
-            return Exception("String is not ascii!")
+        if isinstance(param, str) and param.isascii():
+            return param
+        return Exception("String is not ascii!")
 
 
-def validate_alphabetical(form_input, error_key, error_dict=None):
-    if form_input.isalpha():
+def validate_isalpha(form_input):
+    # валидация являются ли вводимые значения символами
```

Comment:
For the function documentation it would be better to use multi-docstring:
```
def validate_isalpha(form_input):
    """ 
    валидация являются ли вводимые значения символами
    """